### DISCUSSION TITLE
Help Request: [problem with building the project]

### DISCUSSION THREAD

**Discussion by FanisBak** [TEAMMATES/teammates#13025] [Help requests]
### Intention

While giving the command ". /gradlew serverRun & " the back - end server is stuck at 85% executing. I saw in the documentation (https://teammates.github.io/teammates/troubleshooting-guide.html) at the Troubleshooting page that it is expected if you are running the server in the foreground, but I do not know what to do so that the project gets successfully built. 
I also get this:

* What went wrong:
Execution failed for task ':serverRun'.
> Process 'command '/Library/Java/JavaVirtualMachines/jdk-11.jdk/Contents/Home/bin/java'' finished with non-zero exit value 1
BUILD FAILED in 5s
6 actionable tasks: 3 executed, 3 up-to-date

If anyone could help, it would be much appreciated.
Thanks!

**Comment by zedonggg** [comment]
Replying to this a year later for anyone who might be encountering a similar issue:

Running the server gave me a similar execution failed error for serverRun with the detailed logs along the lines of not being able to bind to port 8080. 

Running `lsof -i :8080` on macOS (`netstat` on Windows) showed a process on the 8080 port, which I then terminated using `kill <PID>`. Re-running `./gradlew serverRun` ran successfully.
